library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/AbstractPojoJoin.java

```diff
@@ -47,9 +50,14 @@
   protected Class<?> outClass;
   private transient Map<String,PojoUtils.Getter> gettersStream1;
   private transient Map<String,PojoUtils.Getter> gettersStream2;
-  private transient Map<String,PojoUtils.Setter> setters;
+  protected transient Map<String,PojoUtils.Setter> setters;
+  protected transient Map<String, KeyValPair<STREAM, String>> outputToInputMap;
   protected transient String[] leftKeys;
   protected transient String[] rightKeys;
+  protected enum STREAM
```

Comment:
I think this enum should be public. Otherwise how would anyone access this from outside of this package?